src/__tests__/services/baseClient.test.ts

```diff
@@ -0,0 +1,3 @@
+import BaseClient from '../../services/baseClient/baseClient';
+
+const bc = new BaseClient();

```

Comment:
'bc' is assigned a value but never used.eslint@typescript-eslint/no-unused-vars

Reply:
Må slettas